__mocks__/styles.mock.tsx

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
even better would be to `identity-obj-proxy` which will not leave all the css classes as `undefined` and wouldn't have to be mocked for each css file. See https://github.com/keyz/identity-obj-proxy